nixos/packages.tt

```diff
@@ -153,6 +153,18 @@ function refilter() {
   updateTable();
 };
 
+// Given a license object, or array of license objects, produce suitable
+// HTML for it.
+function licenseHTML(license) {
+  if (typeof license == 'string') {
+    return license;
+  }
+  if (Array.isArray(license)) {
+    return license.map(licenseHTML).join(', ');
+  }
+  return '<a href="' + license.url + '">' + license.spdxId + '</a>';
```

Comment:
It's probably a good idea to handle the case where `url` or `spdxId` are missing, and to escape them.